root/documentation/overlap.conf

```diff
@@ -52,7 +52,7 @@
       </so_term>
       <variant_set>
         type=String
-        description= Short name of a set to restrict the variants found. (<a target="_blank" href="http://www.ensembl.org/info/genome/variation/data_description.html\#variation_sets">See list of short set names</a>) 
+        description= Short name of a set to restrict the variants found. (<a target="_blank" href="//www.ensembl.org/info/genome/variation/species/sets.html">See list of short set names</a>) 
```

Comment:
Why was the protocol dropped from the href tag? Would that not break it?

Reply:
Normally it should pick up the default protocol (i.e. httpS for Ensembl).